sdk/kas_client.go

```diff
@@ -20,25 +22,21 @@ const (
 	secondsPerMinute = 60
 )
 
-type RequestBody struct {
-	KeyAccess       `json:"keyAccess"`
-	ClientPublicKey string `json:"clientPublicKey"`
-	Policy          string `json:"policy"`
-}
-
 type KASClient struct {
 	accessTokenSource auth.AccessTokenSource
 	dialOptions       []grpc.DialOption
 	sessionKey        *ocrypto.RsaKeyPair
 }
 
-// once the backend moves over we should use the same type that the golang backend uses here
-type rewrapRequestBody struct {
-	KeyAccess       KeyAccess `json:"keyAccess"`
-	Policy          string    `json:"policy,omitempty"`
-	Algorithm       string    `json:"algorithm,omitempty"`
-	ClientPublicKey string    `json:"clientPublicKey"`
-	SchemaVersion   string    `json:"schemaVersion,omitempty"`
+type KAOResult struct {
+	SymmetricKey      []byte
+	Error             error
+	KeyAccessObjectID string
+}
+
+type Decryptor interface {
```

Comment:
Only export when there is a specific need to use the type in callers. This is a package internal type and should not be exported
```suggestion
type decryptor interface {
```